pkg/server/config.go

```diff
@@ -386,7 +386,7 @@ func MakeConfig(ctx context.Context, st *cluster.Settings) Config {
 		ScanInterval:                   defaultScanInterval,
 		ScanMaxIdleTime:                defaultScanMaxIdleTime,
 		EventLogEnabled:                defaultEventLogEnabled,
-		EnableWebSessionAuthentication: defaultEnableWebSessionAuthentication,
+		EnableWebSessionAuthentication: envutil.EnvOrDefaultBool("COCKROACH_REQUIRE_WEB_LOGIN", false),
```

Comment:
Move this to a top-level var and reference it here.

Reply:
Done.